README.md

```diff
@@ -1243,6 +1246,305 @@ When you want info about the current school year add /huidig to the url
 
 ---
 
+### vak keuzes: `GET /rest/v1/vakkeuzes`
+
+Fetches all the subjects you are currently enrolled in.
+
+#### Parameters
+
+| Name          | Type   | Value                 |
+|---------------|--------|-----------------------|
+| Authorization | Header | Bearer [access_token] |
+| additional    | Parameter | vaknormering       |
+| additional    | Parameter | actiefOpPeildatum  |
+
+#### Returns
+
+```json
+{
+  "items": [
+    {
+      "$type": "onderwijsinrichting.RVakkeuze",
+      "links": [
+        {
+          "id": xxxxxxxxxx,
+          "rel": "self",
+          "type": "onderwijsinrichting.RVakkeuze",
+          "href": "https://api.somtoday.nl/rest/v1/vakkeuzes/xxxxxxxxxx"
+        }
+      ],
+      "permissions": [
+        {
+          "full": "onderwijsinrichting.RVakkeuze:READ:INSTANCE(xxxxxxxxxx)",
+          "type": "onderwijsinrichting.RVakkeuze",
+          "operations": [
+            "READ"
+          ],
+          "instances": [
+            "INSTANCE(xxxxxxxxxx)"
+          ]
+        }
+      ],
+      "additionalObjects": {
+        "vaknormering": {
+          "$type": "onderwijsinrichting.RVakNormering",
+          "vakId": yyyyyyyyyy,
+          "toetsnormering1": "Standaard",
+          "toetsnormering2": "Alternatief"
+        }
+      },
+      "vak": {
+        "links": [
+          {
+            "id": yyyyyyyyyy,
+            "rel": "self",
+            "type": "onderwijsinrichting.RVak",
+            "href": "https://api.somtoday.nl/rest/v1/vakken/yyyyyyyyyy"
+          }
+        ],
+        "permissions": [
+          {
+            "full": "onderwijsinrichting.RVak:READ:INSTANCE(yyyyyyyyyy)",
+            "type": "onderwijsinrichting.RVak",
+            "operations": [
+              "READ"
+            ],
+            "instances": [
+              "INSTANCE(yyyyyyyyyy)"
+            ]
+          }
+        ],
+        "additionalObjects": {},
+        "afkorting": "ne",
+        "naam": "Nederlandse taal"
+      }
+    },
+    ...
+  ]
+}
+```
+
+---
+
+
+### waarnemingen: `GET /rest/v1/waarnemingen`
+
+Fetches all the waarnemingen currently tied to your account, filter them by date, isGeoorloofd and/or waarnemingSoort.
```

Comment:
Maybe it would be neater if `waarnemingen` was translated into English

Reply:
het zijn jouw docs ga je gang, ik weet niet of 'observations' het duidelijker maakt wat de endpoint doet, maar dat is aan jouw.

Reply:
Ik weet eigenlijk ook niet goed hoe je dit zou moeten vertalen.

Reply:
Even ter zijde, moet ik bij de andere gwn op resolve conversation klikken of moet ik nog iets aanpaseen, want dan weet ik niet wat.

Reply:
Als het lukt zou ik graag zien dat je de formatting van de tabellen aanpast zodat alle velden in de kolommen netjes onder elkaar staan.

Reply:
Ik heb hget aangepast, behalve de vertaling van `waarneming` .